### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by gabrielsroka**
minor corrections

### CODE REVIEW

File: packages/@okta/vuepress-site/reference/okta_expression_language/index.md [hunk 1]
```diff
@@ -255,12 +255,12 @@ The format for conditional expressions is
...
-* The OR operator
-* The ! operator to designate NOT
-* Standard arithmetic operators including <code>&lt;</code>, <code>&gt;</code>, <code>&lt;=</code>, and <code>&gt;=</code>.
+* The <code>AND</code> operator
+* The <code>OR</code> operator
+* The <code>!</code> operator to designate NOT
+* Standard relational operators including <code>&lt;</code>, <code>&gt;</code>, <code>&lt;=</code>, and <code>&gt;=</code>.
 
-**Note:** Use the double equals sign, <code>==</code>, to check for equality.
+**Note:** Use the double equals sign, <code>==</code>, to check for equality and </code>!=</code> for inequality.
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/reference/okta_expression_language/index.md]: @gabrielsroka can we just use the Markdown code formatting? 

e.g. 
```
`AND` `OR` `!`
```

Not sure why someone used HTML formatting here.

**Comment by gabrielsroka** [comment]
Sure. I was just copying what was already there

**Comment by nbarbettini** [comment]
@gabrielsroka Once you've updated the PR with the changes Jakub described, I'll merge this in.

**Comment by nbarbettini** [comment]
Thanks @gabrielsroka! Superseded by https://github.com/okta/okta-developer-docs/pull/455

File: packages/@okta/vuepress-site/reference/okta_expression_language/index.md (hunk 1 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/reference/okta_expression_language/index.md]: Just for the record, it turned out that without the HTML formatting, the site generator did not produce the correct symbols. There's some kind of glitch in rendering the markdown for this. So, in https://github.com/okta/okta-developer-docs/pull/462 Gabriel reverted to the HTML formatting.

**Pull request closed without merging**